helm/bold-common/README.md

```diff
@@ -163,6 +178,35 @@ _See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command doc
 
 ## Application Startup
 
-Configure the Bold BI application startup to use the application. Please refer the following link for more details on configuring the application startup.
+1. Configure the Bold BI application startup to use the application. Please refer the following link for more details on configuring the application startup.
```

Comment:
1. Configure the Bold BI application startup to use the application. Please refer to the following link for more details on configuring the application startup.

Reply:
concern addressed